src/components/Tabset/pageObject/index.js

```diff
@@ -26,6 +26,60 @@ class PageTabset {
         }
         return null;
     }
+
+    /**
+     * Returns true when the left button existing.
+     * @method
+     * @returns {bool}
+     */
+    isLeftButtonExisting() {
+        return !!$(this.rootElement).$$('.rainbow-tabset_button-icon')[0];
+    }
+
+    /**
+     * Returns true when the right button existing.
+     * @method
+     * @returns {bool}
+     */
+    isRightButtonExisting() {
+        return !!$(this.rootElement).$$('.rainbow-tabset_button-icon')[1];
+    }
+
+    /**
+     * Returns true when the left button is enabled.
+     * @method
+     * @returns {bool}
+     */
+    isLeftButtonEnabled() {
+        return $(this.rootElement).$$('.rainbow-tabset_button-icon')[0].isEnabled();
+    }
+
+    /**
+     * Returns true when the right button is enabled.
+     * @method
+     * @returns {bool}
+     */
+    isRightButtonEnabled() {
+        return $(this.rootElement).$$('.rainbow-tabset_button-icon')[1].isEnabled();
+    }
+
+    /**
+     * Click the left button.
+     * @method
+     * @returns {bool}
+     */
+    clickToLeftButton() {
+        return $(this.rootElement).$$('.rainbow-tabset_button-icon')[0].click();
```

Comment:
since the selector '.rainbow-tabset_button-icon' is used many times put in a constant
```BUTTON_SELECTOR = '.rainbow-tabset_button-icon';```